cfgov/jinja2/v1/_includes/organisms/item-introduction.html

```diff
@@ -0,0 +1,81 @@
+{# ==========================================================================
+
+   Item Introduction
+
+   ==========================================================================
+
+   Description:
+
+   Create an Item Introduction molecule.
+   See [GHE]/flapjack/Modules-V1/wiki/Item-Introduction
+
+   value:           An object with the following options for value.
+
+   value.category_label     The category label text.
+   value.title_heading:     Heading text.
+   value.paragraph_heading: Body introduction text.
+   value.authors:           Array of author names and associated URLs.
+
+   value.date:              A datetime for the post.
+   value.has_social:        Whether to show the share icons or not.
+
+   ========================================================================== #}
+
+{% if not value %}
+    {% set lipsumText = lipsum(1, false, 10, 25) | safe %}
+    {% do global_dict.update(
+        {
+            'intro_introduction': {
+                'category_label': 'info for consumers',
+                'title_heading': 'Item Introduction',
+                'paragraph_heading': lipsumText,
+                'authors': [
+                              {'text':'Pat Govning', 'url':'#'},
+                              {'text':'Ingrid Phed', 'url':'#'}
+                          ],
+                'date': '12-12-2015',
+                'has_social': true
+            }
+        })
+    %}
+    {% set value = global_dict.intro_introduction %}
+{% endif %}
+
+{% import 'molecules/social-media.html' as social_media with context %}
+{% import 'macros/category-slug.html' as category_slug %}
+{% import 'macros/time.html' as time %}
+<div class="o-item-introduction">
+    <div class="h4">
+        {{ category_slug.render('info for consumers') }}
+    </div>
+
```

Comment:
Just spaced out the header, but not needed. Fixed in https://github.com/cfpb/cfgov-refresh/commit/5d7ad5b73e4cb954e301f8d3fa4ddb5e1e7dc000